### PR TITLE
SC-3763 (update DoC visit alerts)

### PR DESCRIPTION
**Pull request by enrjp**
This pull request updates the Alerts available to DOC staff that can be issued during a call.

Per Liz's instructions, I also updated the styling on the send confirmation dialog itself to make it a bit clearer to the user what they are sending. Below, you can see how it has changed.

Before:
Screen Shot 2022-10-19 at 11 43 32 AM

After:
Screen Shot 2022-10-19 at 11 42 16 AM

### CODE REVIEW

File: src/components/ConnectionDetailDrawer/index.tsx [hunk 1]
```diff
@@ -313,7 +313,7 @@ const ConnectionDetailDrawer: React.FC<Props> = ({
               label="Allowed messaging"
               labelStyle={labelStyle}
             >
-              {eMessagingRestriction ? "No" : "yes"}
+              {eMessagingRestriction ? "No" : "Yes"}
```
Comment (enrjp) [inline src/components/ConnectionDetailDrawer/index.tsx:316]: Noticed all other permissions indicators were capitalized except this one, so I fixed

File: src/pages/LiveCallPage/VideoCall.tsx [hunk 2]
```diff
@@ -53,7 +53,7 @@ const VideoCall: React.FC<Props> = React.memo(
       try {
         await connectCall.sendMessage(alert.body);
         openNotificationWithIcon(
-          "Alert succesfully issue.",
+          "Alert succesfully issued.",
```
Comment (enrjp) [inline src/pages/LiveCallPage/VideoCall.tsx:56]: Noticed a small grammatical issue, so I fixed

File: src/pages/LiveCallPage/VideoOverlay.tsx [hunk 3]
```diff
@@ -91,12 +91,12 @@ export default function VideoOverlay({
       </Dropdown>
       <Typography.Text></Typography.Text>
       <Modal
-        title="Are you sure you want to send this alert?"
+        title="Send the following alert message?"
```
Comment (enrjp) [inline src/pages/LiveCallPage/VideoOverlay.tsx:94]: Updated copy per Liz's instructions

File: src/pages/LiveCallPage/VideoOverlay.tsx [hunk 4]
```diff
@@ -91,12 +91,12 @@ export default function VideoOverlay({
...
         visible={!!selectedAlert}
+        okText="Send"
```
Comment (enrjp) [inline src/pages/LiveCallPage/VideoOverlay.tsx:96]: Updated button copy per Liz's instructions

File: src/pages/LiveCallPage/VideoOverlay.tsx [hunk 5]
```diff
@@ -91,12 +91,12 @@ export default function VideoOverlay({
...
         onOk={handleConfirmation}
         onCancel={() => setSelectedAlert(null)}
       >
-        <p>Send the following alert to the participants:</p>
```
Comment (enrjp) [inline src/pages/LiveCallPage/VideoOverlay.tsx:99]: Removed this from the body per Liz's instructions

File: src/typings/Call.ts [hunk 6]
```diff
@@ -1,8 +1,6 @@
 export type GridOption = 1 | 2 | 4 | 6 | 8;
 
 export interface CallAlert {
-  id: number;
-  title: string;
```
Comment (enrjp) [inline src/typings/Call.ts:5]: No longer need these fields

File: src/utils/constants/Common.ts [hunk 7]
```diff
@@ -44,65 +44,28 @@ export const GRID_TO_VH_HEIGHT: { [key in GridOption]: number } = {
...
-    body: "Please be patient, your incarcerated contact is joining the call soon. Your call will begin shortly.",
-  },
-  {
-    id: 11,
-    title: "Standby Technical Difficulties",
-    body: "Please be patient, we are working to fix some technical difficulties. Your call will resume shortly.",
-  },
-];
+export const CALL_ALERTS: Record<string, string> = {
+  "Unallowed visitor":
```
Comment (enrjp) [inline src/utils/constants/Common.ts:48]: Per SC ticket instructions, remove ID field from data and use the title as unique key

File: src/pages/LiveCallPage/VideoOverlay.tsx [hunk 8]
```diff
@@ -49,9 +49,9 @@ export default function VideoOverlay({
   const menu = (
     <Menu>
       <SubMenu key="alert" icon={<MessageFilled />} title="Send Alert">
-        {alerts.map((alert) => (
-          <Menu.Item key={alert.id} onClick={() => setSelectedAlert(alert)}>
-            {alert.title}
+        {Object.entries(alerts).map(([title, body]) => (
```
Comment (enrjp) [inline src/pages/LiveCallPage/VideoOverlay.tsx:52]: alerts is now an object, not a list of objects

**Review by wittejm — APPROVED** [review]

File: src/utils/constants/Common.ts [hunk 9]
```diff
@@ -44,65 +44,24 @@ export const GRID_TO_VH_HEIGHT: { [key in GridOption]: number } = {
...
+    "Any use or display of drugs, alcohol, related paraphernalia, or intoxication is strictly prohibited.",
+  "Profanity/hostile language":
+    "Any ongoing use of profanity or loud/hostile communication is prohibited.",
+  "Show photo ID":
+    "To visitors on the call — please show your photo ID to the camera and ensure your face is visible. This is required for the call to continue. Thank you.",
+  "Standby: call beginning soon":
+    "Your contact will be joining shortly. Thank you for your patience.",
+  "Standby: technical difficulties":
+    "Facility staff is working to fix some technical difficulties we detected. We apologize for the inconvenience, and your call will resume shortly.",
+};
```
Comment (enrjp) [inline src/utils/constants/Common.ts:67]: Specific copy changes can be referenced in the Google Sheet linked in the Shortcut ticket

File: src/pages/LiveCallPage/VideoOverlay.tsx (hunk 8 above)
Comment (enrjp) [reply src/pages/LiveCallPage/VideoOverlay.tsx:52]: I agree, I did like that as well. I can make some quick updates 👍🏻

**Pull request merged**